build/Common.props

```diff
@@ -28,7 +28,7 @@
     <MicrosoftExtensionsHostingAbstractionsPkgVer>[2.1.0,5.0)</MicrosoftExtensionsHostingAbstractionsPkgVer>
     <MicrosoftExtensionsOptionsPkgVer>[3.1.0,)</MicrosoftExtensionsOptionsPkgVer>
     <MicrosoftNETFrameworkReferenceAssembliesPkgVer>[1.0.0,2.0)</MicrosoftNETFrameworkReferenceAssembliesPkgVer>
-    <MicrosoftOwinPkgVer>[4.1.1]</MicrosoftOwinPkgVer>
+    <MicrosoftOwinPkgVer>[4.2.2,)</MicrosoftOwinPkgVer>
```

Comment:
Keep this as `[4.1.1,5.0)`? The instrumentation package doesn't really need `4.2.2`. So, if someone wanted to use it with `4.1.1` they still can. Also, 5.0+ could possibly bring in breaking changes so let's wait till it's released and see if the instrumentation library needs any changes.

Reply:
<4.2.2 has a [vulnerability](https://github.com/advisories/GHSA-3rq8-h3gj-r5c6) so I'd rather not propagate usage of that version of the package.

Reply:
@utpilla It looks like a [security thing](https://github.com/advisories/GHSA-3rq8-h3gj-r5c6) so I'm good with bumping to `4.2.2`. Regarding if we should have the upper bound, I'm good with that if that's what you want to do!

Reply:
Okay. That's good! I wasn't aware of it.